DirectX11Proj/IScene.h

```diff
@@ -8,5 +13,7 @@ class IScene
 	virtual void Tick() = 0;
 	virtual void Init() = 0;
 	virtual void Destroy() = 0;
+
+	std::vector<IObject> mObjects;
```

Comment:
especially because objects can dynamically be added - what if you have 1000 objects and another one has to be added?